test/sslcorrupttest.c

```diff
@@ -196,6 +199,29 @@ static int test_ssl_corrupt(int testidx)
         goto end;
     }
 
+    ciphers = SSL_CTX_get_ciphers(cctx);
+    if (ciphers == NULL || sk_SSL_CIPHER_num(ciphers) != 1) {
+        printf("Unexpected ciphers set\n");
```

Comment:
yeah, but errors should go to stderr.  we waffle:  just doing what's already done vs fix it all while you have the file opened.  we should about a policy.  but like i said, +1 to what you did.